test/functional/reproducibleCompare/linux_repro_build_compare.sh

```diff
@@ -0,0 +1,153 @@
+#!/bin/sh
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+# This script examines the given SBOM metadata file, and then builds the exact same binary
+# and then compares with the Temurin JDK for the same build version, or the optionally supplied TARBALL_URL.
+
+installPrereqs() {
+  if test -r /etc/redhat-release; then
+    yum install -y gcc gcc-c++ make autoconf unzip zip alsa-lib-devel cups-devel libXtst-devel libXt-devel libXrender-devel libXrandr-devel libXi-devel
+    yum install -y file fontconfig fontconfig-devel systemtap-sdt-devel # Not included above ...
+    yum install -y git bzip2 xz openssl pigz which jq # pigz/which not strictly needed but help in final compression
+    if grep -i release.6 /etc/redhat-release; then
+      if [ ! -r /usr/local/bin/autoconf ]; then
+        curl https://ftp.gnu.org/gnu/autoconf/autoconf-2.69.tar.gz | tar xpfz - || exit 1
+        (cd autoconf-2.69 && ./configure --prefix=/usr/local && make install)
+      fi
+    fi
+  fi
+}
+
+# ant required for --create-sbom
+downloadAnt() {
+  if [ ! -r /usr/local/apache-ant-${ANT_VERSION}/bin/ant ]; then
+    echo Downloading ant for SBOM creation:
+    curl https://archive.apache.org/dist/ant/binaries/apache-ant-${ANT_VERSION}-bin.zip > /tmp/apache-ant-${ANT_VERSION}-bin.zip
```

Comment:
Currently it will only enable for weekly dev tests with linux. Shouldn't be a heavy use. @andrew-m-leonard I will leave for you to decide if we need to update this later.

Reply:
I don't think we would hit those limits, the ant zip is not large